### ISSUE TITLE
nyaa.se RSS feed broken in Taiga

### ISSUE THREAD

**Issue by otvenharom** [erengy/taiga#313]
Using the below URL fails with the following:
http://www.nyaa.se/?page=rss&cats=1_37&filter=2

![image](https://cloud.githubusercontent.com/assets/5750392/19216921/6e732512-8dd0-11e6-9b45-93f69715a4f4.png)

The URL opens without any problems in any other browser. Previously it had worked without issues, but in the past few days I noticed that Taiga was no longer downloading torrents for new releases automatically. 

Thanks for reading.

**Comment by erengy** [comment]
Duplicate of #311 and #236.

This happens when Nyaa enables their DDoS protection by Cloudflare:

![nyaa-cloudflare](https://cloud.githubusercontent.com/assets/1078430/19216974/d7f2eb62-8dda-11e6-91d6-b5cf2ed2f526.png)

...which makes their RSS feeds unusable for non-browser applications such as Taiga.

**Comment by otvenharom** [comment]
I see. I suppose only switching to a different RSS would solve this. Strange how it had worked correctly for me until this week, the previous issues are much older than that.

**Comment by erengy** [comment]
It's a switch they flip from time to time, when they're under attack. It seems they switched it off now; I no longer get the error.

As I [mentioned](https://github.com/erengy/taiga/issues/236#issuecomment-188931617), there's no way for Taiga to bypass the security check, unfortunately. The only solution is to choose another RSS feed from *Settings → Torrents → Discovery → Sources*, when Nyaa doesn't respond.

**Comment by onlybrad** [comment]
Nyaa feeds stopped working on taiga today.

**Comment by Artins90** [comment]
I have been experiencing a strange issue since yesterday, the torrents tab stays empty when pressing the "Check new torrents" button or when triggered by the automatic timer.
The rss update seems to work only once every 20-25 tries when pressing the check button repeatedly.
The connectivity to the the rss feed https://nyaa.si/rss?c=1_2&f=0 works fine via Chrome at the first try.
So far I have tried to reinstall Taiga, I have emptied all the cache categories under settings and I have attempted a second install to a different drive to no avail.
